### PR TITLE
Add keywords for and nameonly and remove indentation

### PR DESCRIPTION
**Pull request by mschlaipfer**
Adding new keywords, untested.

### CODE REVIEW

**Comment by mschlaipfer** [comment]
Removing indentation as requested in https://github.com/rouge-ruby/rouge/pull/1647

File: lib/rouge/lexers/dafny.rb [hunk 1]
```diff
@@ -15,12 +15,12 @@ class Dafny < RegexLexer
         break calc case class codatatype const constructor
         datatype decreases default
```
Comment (RustanLeino) [inline lib/rouge/lexers/dafny.rb]: Missing keywords:
* `allocated`
* `by`
* `export`
* `reveal` (in addition to `reveals`, which is already mentioned)
* `witness`

I would suggest also listing the following as keywords, even though--technically--they are only contextual keywords:
* `to`
* `downto`

These are no longer keywords:
* `default`
* `inductive`
* `where`

Missing type:
* `ORDINAL`

File: spec/visual/samples/dafny [hunk 2]
```diff
@@ -87,36 +87,36 @@ set_ if_ as_ is_ int_ true_
 set' if' as' is' int' true'
 
 // Types
-        bool char int real string object object?
-        seq set map iset imap multiset
-        array array2 array10 array?
-        bv0 bv1 bv10 bv9
+bool char int real string object object?
+seq set map iset imap multiset
+array array2 array10 array?
```
Comment (RustanLeino) [inline spec/visual/samples/dafny]: Other good examples to include here are the keywords `array2?` and `array10?`.

File: lib/rouge/lexers/dafny.rb (hunk 1 above)
Comment (mschlaipfer) [reply lib/rouge/lexers/dafny.rb]: done

File: spec/visual/samples/dafny (hunk 2 above)
Comment (mschlaipfer) [reply spec/visual/samples/dafny]: added

**Comment by mschlaipfer** [comment]
Can we merge these changes so the PR to rouge-ruby gets updated?

**Pull request merged**